src/StatefulButton/__snapshots__/StatefulButtontest.test.jsx.snap

```diff
@@ -5,10 +5,9 @@ exports[`StatefulButton renders basic usage 1`] = `
   <button
     aria-disabled={false}
     aria-live="assertive"
-    className="btn pgn__stateful-btn pgn__stateful-btn-state-default btn-primary mr-2"
-    onBlur={[Function]}
+    className="pgn__stateful-btn pgn__stateful-btn-state-default btn-primary mr-2 btn btn-primary"
```

Comment:
btn-primary is listed twice here, as well.

Reply:
Thanks, I think I know why this is the case and will fix.